File: Defanging-an-IP-Address .js
/**
 * @param {string} address
 * @return {string}
 */
var defangIPaddr = function(address) {
 const   ipArray =address.split(".")
 let res = []
   for (let i = 0;i<(ipArray.length);i++){
     res.push(ipArray[i] ,"[.]")
   }
  res.pop()
 return res.join("")
};

File: Defanging-an-IP-Address.py
class Solution:
    def defangIPaddr(self, address: str) -> str:
        solution = ""
        for element in address:
            if element == ".":
                solution +="[.]"
            else:
                solution += element
        return solution
        
#  class Solution:
#     def defangIPaddr(self, address: str) -> str:
#         return address.replace(".","[.]")
